test/DebugInfo/DebugControlFlow.cl

```diff
@@ -11,10 +11,9 @@
 // Otherwise, debug info interferes with SPIRVToLLVM translation
 // of structured flow control
 //
-// Currently, Line DebugInfo instructions are still present
+// Line DebugInfo instructions are still present
 // between LoopMerge and Branch/BranchConditional instructions.
```

Comment:
Is it valid according to the spec?

I'm just curious if everything is valid, why is it so important to mention this fact? Otherwise, we should probably put some TODO here to fix what is incorrect

Reply:
The spec clearly states that `OpLoopMerge` must be second to last in its basic block, `OpBranch` being its immediate/only successor. Moreover, taking another glance at the spec:

"- if a branch merge instruction is used, the last `OpLine` in the block must be before its merge instruction"

I think I should remove the comment here and in the test for LoopContol brought by #313. I should add the TODO in the code that generates `OpLine`s.

Reply:
It remains to be determined why does incorrect `OpLine` placement NOT interfere with backwards translation.